src/devices/oregon_scientific.c

```diff
@@ -142,6 +142,26 @@ static unsigned long long cm180i_total(uint8_t const *msg)
     return val;
 }
 
+static unsigned int swap_nibbles(unsigned char byte) {
+ return (((byte&0xf)<<4) | (byte>>4));
```

Comment:
Indent is 4 spaces please. Add spaces around operators (`) << 4`)

Reply:
Should be resolved